pkg/provider/aad/testdata/LoginEmbeddedJsonExtraJavascript.html

```diff
@@ -0,0 +1,12 @@
+  <meta name="robots" content="none" />
+
+  <script type="text/javascript">//<![CDATA[
+  $Config={"name": "value&amp;with ampersand"}; alert('oops');
```

Comment:
Since your original design used `strings.IndexLast(config, "}")` it shouldn’t have broken on this. This is why I had the `if (false) { … }`, I had to have a `}` after the end of the JSON.

Reply:
true, this test is useless